packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js

```diff
@@ -198,11 +199,16 @@ const links = [
   },
 ];
 
-const DocsLayout = ({ children, ...rest }) => (
-  <SideNavLayout links={links} {...rest}>
-    {children}
-  </SideNavLayout>
-);
+const DocsLayout = ({ children, ...rest }) =>
+  process.env.BPK_NEO ? (
+    <NeoSideNav activeSection="components" links={links} {...rest}>
```

Comment:
I feel like `activeSection` should be derived from the current `location` (from react router). That's how the old side nav works it out...